package-system/expat/build_config.json

```diff
@@ -0,0 +1,143 @@
+{
+    "git_url":"https://github.com/libexpat/libexpat",
+    "git_tag":"R_2_4_2",
+    "package_name":"expat",
+    "package_version":"2.4.2-rev1",
+    "package_url":"https://libexpat.github.io/",
+    "package_license":"MIT",
+    "package_license_file":"expat/COPYING",
+    "cmake_find_source":"Findexpat.cmake",
+    "cmake_find_target":"Findexpat.cmake",
+    "cmake_src_subfolder": "expat",
+    "Platforms":{
+        "Windows":{
+          "Windows": {
+              "custom_cmake_install": true,
+              "cmake_generate_args": [
+                  "-G",
+                  "\"Visual Studio 16 2019\"",
+                  "-DCMAKE_CXX_STANDARD=17",
+                  "-DEXPAT_BUILD_DOCS=OFF",
+                  "-DEXPAT_BUILD_TOOLS=OFF",
+                  "-DEXPAT_BUILD_EXAMPLES=OFF",
+                  "-DEXPAT_SHARED_LIBS=OFF",
+                  "-DEXPAT_BUILD_FUZZERS=OFF",
+                  "-DEXPAT_BUILD_TESTS=OFF",
+                  "-DBUILD_SHARED_LIBS=0"
+              ],
+              "cmake_build_args": [
+                  "--parallel"
+              ],
+              "build_configs": [
+                  "Release"
+              ],
+              "custom_test_cmd" : [
+                "test_expat_windows.cmd"
+              ]
+          },
+          "Android": {
+            "custom_cmake_install": true,
+            "cmake_generate_args_release": [
+                "-G",
+                "Ninja",
+                "-DCMAKE_TOOLCHAIN_FILE=../../../../Scripts/cmake/Platform/Android/Toolchain_android.cmake",
+                "-DCMAKE_CXX_STANDARD=17",
+                "-DEXPAT_BUILD_DOCS=OFF",
+                "-DEXPAT_BUILD_TOOLS=OFF",
+                "-DEXPAT_BUILD_EXAMPLES=OFF",
+                "-DEXPAT_SHARED_LIBS=OFF",
+                "-DEXPAT_BUILD_FUZZERS=OFF",
+                "-DEXPAT_BUILD_TESTS=OFF",
+                "-DBUILD_SHARED_LIBS=0"
+            ],
+            "cmake_build_args": [
+                "--parallel"
+            ],
+            "build_configs": [
+                "Release"
+            ],
+            "custom_test_cmd" : [
+                "test_expat_android.cmd"
+              ]
+        }
+        },
+        "Darwin": {
+          "Mac": {
+              "custom_cmake_install": true,
+              "cmake_generate_args_release": [
+                  "-G",
+                  "Xcode",
+                  "-DEXPAT_BUILD_DOCS=OFF",
+                  "-DEXPAT_BUILD_TOOLS=OFF",
+                  "-DEXPAT_BUILD_EXAMPLES=OFF",
+                  "-DEXPAT_SHARED_LIBS=OFF",
+                  "-DEXPAT_BUILD_FUZZERS=OFF",
+                  "-DEXPAT_BUILD_TESTS=OFF",
+                  "-DCMAKE_TOOLCHAIN_FILE=../../../../Scripts/cmake/Platform/Mac/Toolchain_mac.cmake",
+                  "-DBUILD_SHARED_LIBS=0"
```

Comment:
suggest giving the same ordering as the previous cases so it is easier to compare
if ordering doesnt affect anything (which should be this case), using alphabetical ordering keeps all the lists consistent

Reply:
OK.  I might add some additional changes to a new PR for next time we build expat packages and so that it serves as a good example.

Reply:
sure, I am fine with another PR